clusters/build-clusters/01_cluster/loki.yaml

```diff
@@ -0,0 +1,335 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: loki
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: loki-promtail
+rules:
+- apiGroups:
+  - ''
+  resources:
+  - nodes
+  - nodes/proxy
+  - services
+  - endpoints
+  - pods
+  - configmaps
+  verbs:
+  - get
+  - watch
+  - list
+- apiGroups:
+  - 'config.openshift.io'
+  resources:
+  - 'clusterversions'
+  verbs:
+  - 'get'
+- apiGroups:
+  - security.openshift.io
+  resourceNames:
+  - privileged
+  resources:
+  - securitycontextconstraints
+  verbs:
+  - use
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: loki-promtail
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: loki-promtail
+subjects:
+- kind: ServiceAccount
+  name: loki-promtail
+  namespace: loki
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: loki-promtail
+  namespace: loki
+data:
+  promtail.yaml: |-
+    clients:
+      - backoff_config:
+          max_period: 5m
+          max_retries: 20
+          min_period: 1s
+        batchsize: 102400
+        batchwait: 10s
+        bearer_token_file: /tmp/shared/prod_bearer_token
+        timeout: 10s
+        url: https://observatorium.api.openshift.com/api/logs/v1/dptp/loki/api/v1/push
+    positions:
+      filename: "/run/promtail/positions.yaml"
+    scrape_configs:
+    - job_name: kubernetes
+      kubernetes_sd_configs:
+      - role: pod
+      pipeline_stages:
+      - cri: {}
+      - labeldrop:
+        - filename
+      - pack:
```

Comment:
If we're running this for openshift namespaces only, then cardinality would be fairly low - and `pack`/`labelallow` sections can be dropped

Reply:
Done